csharp/ql/test/library-tests/dataflow/local/LocalDataFlow.cs

```diff
@@ -217,13 +217,13 @@ public void M(bool b)
         // Ad hoc tracking (System.String), tainted
         var sink33 = (string)sink32.Substring(0).ToLowerInvariant().ToUpper().Trim(' ').Replace("a", "b").Insert(0, "").Clone();
         Check(sink33);
-        var sink48 = sink33.Normalize().Remove(4, 5).Split(' ');
```

Comment:
Because `string.Split()` is now modelled (precisely) as a taint step from the qualifier to the elements of the returned array, and local flow is not field/collection sensitive.